### ISSUE TITLE
timeline items with overlapping texts

### ISSUE THREAD

**Issue by yikuanlee** [almende/vis#3441]
The similar issues seemed to be reported at #1417 and #1982  and fixed in v4.20.1.
However, I still see this issue. 
With a certain zoom level, we can see it works with no overlapped. ( one item text is "SoapUI Event" and the other is SoapUI Group Account)
![image](https://user-images.githubusercontent.com/7738862/30342739-df072cd0-97c8-11e7-87d6-3024cf7a9d59.png)
then zoom out a little bit (0.2), we can 2 items text overlapped (
![image](https://user-images.githubusercontent.com/7738862/30342834-371838e2-97c9-11e7-96ff-b8eb9ef77321.png)

It seems like this behavior occurred when one is range item and the other is point item. i expected one will be stacked up to the other.

**Comment by yotamberk** [comment]
Did you remove the `overflow: hidden`?

**Comment by yikuanlee** [comment]
Yes. In order to see full label of range item, the style for item is 
.vis-item .vis-item-overflow {
    overflow: visible;
}

![image](https://user-images.githubusercontent.com/7738862/30445864-6fc8df9c-9955-11e7-9406-4e4b07cd859e.png)

**Comment by yotamberk** [comment]
Unfortunately, once you overwrite the `overflow`, the items will collide. There is no way yet to tell the timeline to allow items to collide by the ranged item width OR the label. 
This could be an easy fix. Want to go get a try at it?

**Comment by yikuanlee** [comment]
if i change back overflow: hidden for range item,  the label for that overlapped item will be disappeared 
(range is not big enough hold label). 
I was wondering if any way we can force to stack up in this situation.

**Comment by yotamberk** [comment]
No easy way to do that. Ranged items were never meant to have overflowing texts. I've seen this issue pop up in the past. I'll leave this issue open.

**Comment by tfsarlo** [comment]
I solved this by using this (https://github.com/brandonaaron/jquery-overlaps) in the point's label attached to the event rangechanged.

**Comment by PauCodina** [comment]
Any idea or comment about this issue?
You can reproduce the malfunction in the [range overflow example](http://visjs.org/examples/timeline/items/rangeOverflowItem.html) of adjusting the zoom to 2013-2017 years.

**Comment by RichRat** [comment]
I have the same Problem after updating from 4.4.0 to 4.20.1
Because all content is going to be direclty or indirectly user generated i have no control over start/end-time nor the length of the label. Overlap is going to happen and disabeling it would make ranges unusable. 
Converting those items to dots would make the timeline pointless since half of the information is missing.
